src/main/resources/mcmod.info

```diff
@@ -1,16 +1,13 @@
 [
 {
   "modid": "extendedcrafting",
-  "name": "Extended Crafting",
-  "description": "Adds some new ways to craft items, as well as extra crafting items and utilities.",
+  "name": "ExtendedCrafting: Omnifactory Edition",
+  "description": "Adds some new ways to craft items, as well as extra crafting items and utilities.\n\nContinued support for 1.12.2 provided by OmnifactoryDevs.",
   "version": "${version}",
   "mcversion": "${mcversion}",
-  "url": "https://minecraft.curseforge.com/projects/extended-crafting",
-  "updateUrl": "",
-  "authorList": ["BlakeBr0"],
-  "credits": "SpitefulFox for Avaritia which some of this mod is based off of, and where the Singularity texture comes from; and ForbiddenMagic, where the Nether Star Block texture comes from.",
-  "logoFile": "",
-  "screenshots": [],
-  "dependencies": []
```

Comment:
Do we want to keep this line and add the Library mod for the original version of Extended Crafting?

Reply:
Since we're not using `useDependencyInformation` the `requiredMods`, `dependencies`, and `dependents` fields in this file would be ignored anyway. We specify the dependency in the `@Mod` annotation in the source code.